components/style/logical_geometry.rs

```diff
@@ -215,6 +215,7 @@ impl Debug for DebugWritingMode {
 
 // Used to specify the logical direction.
 #[derive(Debug, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "servo", derive(Serialize))]
```

Comment:
I fixed this line so Stylo won't derive `Serialize` anymore. Is this safe for Stylo? @Manishearth ?